nix/passthru-vendored.nix

```diff
@@ -26,7 +26,7 @@
           ;
           installPhase = ''
             mkdir -p $out
-            find . -name "*.cdx.json" -execdir install {} $out/{} \;
+            cp *.cdx.json $out/
```

Comment:
This doesn't work as a general solution because it depends on the source code organization of the software in queston. E.g. in Lanzaboote this would fail because there is no "top-level" binary. If you call `cargo-cyclonedx` from `rust/tool` in the Lanzaboote tree, you'll only get `systemd/lzbt-systemd_bin.cdx.json` which wouldn't be picked up by this solution.

Reply:
That's a good point. Lanzaboote is the final boss for the SBOM generation due to the somewhat complicated packaging. But if it works, we can handle everything? :-D

Reply:
There is the possibility to use `cargo build --bins` to receive a list of binaries that will be the result of the build process. I found no other method to retrieve this information. 

Not sure if this is the best solution, but it works for cloud-hypervisor and lanzaboote reliably.

Reply:
I've changed this a bit to avoid relying on Cargo. I tried to find the SBOM for each binary that is actually installed:

https://github.com/blitz/bombon/commit/a00be5cb6deecc8383794ff8ab06acd18f5f05ac

I like this approach, because it will loudly complain if there is a binary that has no SBOM _and_ it will not include dependencies of components that are not installed.

Reply:
Sounds good! :+1:

Reply:
I have switches to julians approach. Will resolve this discussion